### PR TITLE
1.2

### PR DESCRIPTION
**Pull request by tarheel1999**
Changes to categories, RCS Block effects, added action groups to lights,
changed 1x3W solar panel from 1.6 to 3.7 as they mass almost 3 times the
stock unshrouded panels and are physically bigger.

I don't know why git thinks the .cfg files are binaries.

### CODE REVIEW

**Comment by Kerbas-ad-astra** [comment]
Yeah, you got it this time.  I'll check out the panel size tonight and merge or suggest panel balance changes then.

I think the binary-file problem is down to encoding.  TortoiseGit is smart enough to understand they're text files, but it looks like GitHub doesn't like non-UTF-8 text files.  I might see about changing that as a final pass, but that may be seen by GitHub as a mass deletion-and-replacement of a file's contents, so I'll save that till after the revision is done.

**Comment by Kerbas-ad-astra** [comment]
Clockwise from the left, that's the stock 3x2, Ven's 2x3, Ven's 1x3, and the stock 1x6.  I think it's clear that the chargeRate of the 2x3 panel should stay the same.  In that case, It should also match the costs and mass of the 1x6 and 3x2 panels (we can just delete the TechRequired, entryCost, cost, and mass lines from its entry in Extra_Electrical.cfg).

screenshot6

File: GameData/VenStockRevamp/Squad/Parts/Engines.cfg [hunk 1]
```diff
@@ -197,6 +197,21 @@
 	MODEL {
         model = VenStockRevamp/Squad/Parts/Propulsion/RCS-105
 	}
+	{
```
Comment (Kerbas-ad-astra) [inline GameData/VenStockRevamp/Squad/Parts/Engines.cfg]: This guy is extra.

**Comment by tarheel1999** [comment]
The binary thing is weird as I re-encoded those files as UTF-8 but it still didn't like them. Anyway changes made. The adding of the newline was by Github as I used the website to make the changes you requested.

**Pull request merged**